cypress/integration/appTest.js

```diff
@@ -0,0 +1,26 @@
+describe('App Home Page', () => {
+
+  beforeEach(() => {
+    cy.fixture('multipleCities.json').as('cities').then((cities) => {
+      cy.intercept('GET', 'https://softwhere-api.herokuapp.com/cities', {body: cities});
+    });
+
+    cy.visit('http://localhost:3000');
+  });
+
+  it('User should see a header containing the app title and description', () => {
+    cy.get('header').first().contains('softWhere?');
+    cy.get('header').last().contains('Best US Cities to Live as a Software Developer');
+  });
```

Comment:
@sarahrudy just changed the header to ALL CAPS, I wonder if Cypress is case-sensitive?